Makefile

```diff
@@ -31,6 +31,15 @@ ifeq ($(shell test -d $(addprefix $(OR_PREFIX), openssl111) && echo -n yes), yes
 	OPENSSL_PREFIX=$(addprefix $(OR_PREFIX), openssl111)
 endif
 
+ifeq ($(UNAME), Darwin)
+ifeq ($(shell test -d /usr/local/opt/openresty-openssl && echo yes), yes)
+	OPENSSL_PREFIX=/usr/local/opt/openresty-openssl
+endif
+ifeq ($(shell test -d /usr/local/opt/openresty-openssl111 && echo yes), yes)
```

Comment:
Should try this first.

Reply:
@tokers 
if we try `/usr/local/opt/openresty-openssl111` first, and `/usr/local/opt/openresty-openssl` exists, it will overwrite `/usr/local/opt/openresty-openssl111`.

Reply:
Sorry I didn't capture your mean.

Reply:
I think @nic-chen's explanation is clear.
This situation is `if if ` not `if else`.

Reply:
OK, I got it.